core/jobs.py

```diff
@@ -33,8 +33,8 @@
 from mapreduce import context
 from mapreduce import input_readers
 from mapreduce import mapreduce_pipeline
-from mapreduce import model as mapreduce_model
 from mapreduce import output_writers
+from mapreduce import model as mapreduce_model
```

Comment:
@giritheja the import order seems to be incorrect. Could you please fix it? Thanks.

Reply:
<img width="586" alt="screen shot 2018-03-15 at 11 33 20 am" src="https://user-images.githubusercontent.com/8845039/37484067-b9311a54-2844-11e8-82b0-c8d786084eeb.png">
I get the exact opposite error when I try to make changes and push. :/

Reply:
@giritheja Try merging the latest version of upstream/develop into this branch? I think @apb7 made changes here a little while back.

(@apb7, any thoughts?)

Reply:
<img width="759" alt="screen shot 2018-03-15 at 10 30 44 pm" src="https://user-images.githubusercontent.com/8845039/37505179-a3f19748-28a0-11e8-8224-982d2fda76e7.png">
Merged and trying to push but still facing same issue.

Reply:
Hm, I'll have to defer you to @apb7 then. @apb7, could you help please?

(Also, @giritheja, thank you for checking!)

Reply:
@apb7 -- ping?

Reply:
Sorry for the delay, @seanlip! Looking into this.

Reply:
Hi @giritheja, we recently changed the import-order in this file. Did you follow the steps as stated in this [discussion](https://groups.google.com/forum/#!topic/oppia-dev/mVzwx0TDip0)?  
The steps are as follows: 
1. Run `bash scripts/clean.sh`
2. Run `bash scripts/start.sh`
3. Run `python scripts/pre_commit_linter.py --path=./core/jobs.py`

Since you have already pulled the latest version of `develop` branch, isort should not report any errors now. This would solve the issue.  
(@seanlip: If this does not help, I will look into this further!)
Thanks!

Reply:
I just tried this (for another PR) and it works -- thanks @apb7. It did result in an additional lint error in statistics/gae_models but I've fixed that (see https://github.com/oppia/oppia/pull/4818/commits/b7af371fba65c00d7be4331c38a98f0298239d05).

Reply:
Thank you @seanlip! Re [`b7af371`](https://github.com/oppia/oppia/commit/b7af371fba65c00d7be4331c38a98f0298239d05), I think this is the only way to deal with it :)